<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>TabDiskUsage</class>
    <widget class="QWidget" name="TabDiskUsage">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>904</width>
                <height>212</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
            <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                    <property name="spacing">
                        <number>6</number>
                    </property>
                    <item>
                        <spacer name="verticalSpacer_2">
                            <property name="orientation">
                                <enum>Qt::Vertical</enum>
                            </property>
                            <property name="sizeType">
                                <enum>QSizePolicy::Fixed</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>20</width>
                                    <height>45</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <layout class="QGridLayout" name="gridLayout">
                            <property name="leftMargin">
                                <number>20</number>
                            </property>
                            <property name="rightMargin">
                                <number>20</number>
                            </property>
                            <item row="2" column="1">
                                <widget class="QLabel" name="label_space_utilized">
                                    <property name="text">
                                        <string>Space utilized:</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="3">
                                <spacer name="horizontalSpacer_5">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item row="3" column="6" alignment="Qt::AlignLeft">
                                <widget class="ColorButton" name="color_button_space_free" native="true"/>
                            </item>
                            <item row="2" column="5">
                                <widget class="QLabel" name="label_space_utilized_value_in_bytes">
                                    <property name="text">
                                        <string>TextLabel</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="1" colspan="4">
                                <widget class="QComboBox" name="combobox_devices"/>
                            </item>
                            <item row="3" column="7">
                                <widget class="QLabel" name="label_space_total_value">
                                    <property name="text">
                                        <string>TextLabel</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="0">
                                <spacer name="verticalSpacer_3">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Preferred</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item row="1" column="2">
                                <spacer name="horizontalSpacer_2">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Preferred</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item row="2" column="6" alignment="Qt::AlignLeft">
                                <widget class="ColorButton" name="color_button_space_utilized" native="true"/>
                            </item>
                            <item row="0" column="0">
                                <spacer name="horizontalSpacer_3">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item row="1" column="4">
                                <spacer name="horizontalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Preferred</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item row="3" column="3">
                                <widget class="QLabel" name="label_space_free_value">
                                    <property name="text">
                                        <string>TextLabel</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="3" column="5">
                                <widget class="QLabel" name="label_space_free_value_in_bytes">
                                    <property name="text">
                                        <string>TextLabel</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="3" column="1">
                                <widget class="QLabel" name="label_space_free">
                                    <property name="text">
                                        <string>Space free:</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="3">
                                <widget class="QLabel" name="label_space_utilized_value">
                                    <property name="text">
                                        <string>TextLabel</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="6">
                                <spacer name="horizontalSpacer_7">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item row="1" column="8">
                                <spacer name="horizontalSpacer_6">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item row="0" column="7" rowspan="3">
                                <widget class="ChartPie" name="chart_pie" native="true"/>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <spacer name="verticalSpacer">
                            <property name="orientation">
                                <enum>Qt::Vertical</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>20</width>
                                    <height>40</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>ColorButton</class>
            <extends>QWidget</extends>
            <header>.widget_color_pickup</header>
            <container>1</container>
        </customwidget>
        <customwidget>
            <class>ChartPie</class>
            <extends>QWidget</extends>
            <header>.widget_chartpie</header>
            <container>1</container>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
